### DISCUSSION TITLE
Is there a reason why dbeaver only allows one query at a time while pgadmin allows multiple?

### DISCUSSION THREAD

**Discussion by josepgoyl74** [dbeaver/dbeaver#10793] [General]
So I decided to download dbeaver by recommendation of the internet over the default pgadmin

The thing is, even when working on different tables and different SQL editors, beaver seems to lock into only one query: I can't even do a simple "select * from mytable limit 1" on a completely ifferent table [https://9apps.ooo/](url)

Meanwhile on pgadmin I can use the pg stat activity to see my dbeaver query running and I can run as many queries as I want

Is there a reason for this or am I doing it wrong? I remember using dbeaver fine for a week before this began happening.

**Comment by p91paul** [comment]
There is a setting "separate connection for each editor" somewhere in preferences.
All table editors will continue to be in the same connection, blocking each other. However at least sql scripts will live indipendently (and usually the long queries are the ones written is sql editors).
